test/karma-test-shim.js

```diff
@@ -52,6 +52,7 @@ System.config({
     '@angular/cdk/a11y': 'dist/packages/cdk/a11y/index.js',
     '@angular/cdk/bidi': 'dist/packages/cdk/bidi/index.js',
     '@angular/cdk/coercion': 'dist/packages/cdk/coercion/index.js',
+    '@angular/cdk/collections': 'dist/packages/cdk/collections/index.js',
```

Comment:
You can tell, adding this to every system configuration is annoying. 
<sub>I'm thinking about a more dynamic approach here</sub>